dynamic-plugin/src/components/GenieWidgetsPage.tsx

```diff
@@ -41,12 +41,37 @@ function DashboardLayout() {
     <div style={{ padding: '20px' }}>
       {activeDashboard && activeDashboard.layout && (
         <div style={{ marginBottom: '20px' }}>
-          <h2 style={{ margin: '0 0 10px 0', fontSize: '24px', fontWeight: 'bold' }}>
-            {activeDashboard.layout.name || 'Untitled Dashboard'}
-          </h2>
-          <p style={{ margin: '0 0 20px 0', color: '#666', fontSize: '14px' }}>
-            {activeDashboard.layout.description || 'No description available'}
-          </p>
+          <EditableInline
+            value={activeDashboard.layout.name || 'Untitled Dashboard'}
+            isTitle
+            onConfirm={async (updatedTitle) => {
```

Comment:
Can you create a single function to handle both fields and pass it to onConfirm?